driftbase/auth/authenticate.py

```diff
@@ -252,6 +252,8 @@ def authenticate(username, password, automatic_account_creation=True, fallback_u
         "player_uuid": player_uuid.hex if player_uuid else None,
         "roles": user_roles,
     }
+    if my_identity.identity_type == "ethereum":
+        ret["ethereum_address"] = my_user_name.split(":")[1]
```

Comment:
As mentioned offline, I think it's better the entry is called `provider_id` or something along those lines so that for all platforms where we do expose it, it's always available under the same key. The format is always `provider:id` so apps can still determine what type it is.

Reply:
branch updated with a generalization that always provides this value for all providers, with a special case for user+pass where we use the user_id as provider_id

Reply:
Let's use provider_user_id, as it's the user_id from the provider, external or internal